public/index.php

```diff
@@ -25,12 +25,16 @@
 ini_set("display_errors", 1);
 session_start();
 header('Content-Type: text/html; charset=utf-8');
+//Sanitize $_POST parameters
+$post_input  = filter_input_array(INPUT_POST, FILTER_SANITIZE_STRING);
 
 $templates = new League\Plates\Engine('../templates');
 
+$post_input['oauth_consumer_key'] = filter_input(INPUT_POST,'oauth_consumer_key',FILTER_SANITIZE_NUMBER_INT);
```

Comment:
It looks like oauth_consumer_keys are not limited to integers - in fact I don't think the spec defines the character set at all